keyboards/dz60/keymaps/via/rules.mk

```diff
@@ -0,0 +1,3 @@
+MOUSEKEY_ENABLE = no
```

Comment:
I'd like to at least see if this compiles to a workable size in Travis CI without Mouse Keys disabled, as that's a behaviour change from the default keymap.

```suggestion
```